share/models/core.py

```diff
@@ -75,6 +81,37 @@ def create_robot_user(self, username, robot, long_title='', home_page=''):
         return user
 
 
+class FaviconImage(models.Model):
+    user = models.OneToOneField('ShareUser', on_delete=DATABASE_CASCADE, primary_key=True)
```

Comment:
Let django handle the primary key. Things have a tendency to get weird when you don't.